### ISSUE TITLE
OL: Create a map geometry with a rubber band tool

### ISSUE THREAD

**Issue by USGSsakins** [USGS-Astrogeology/CartoCosmos#10]
As a user of a web map interface, I want to create a geometry onto the map by dragging a point between opposite corners of a rectangle:

For the definition of "Done", we are looking for an HTML (or Python notebook) demonstration of the following capabilities:
- [ ] There is a "tool" enable in the OpenLayers web map interface to choose to "draw" a rectangle
- [ ] A geometry can be created on top of the map interface by clicking, then dragging from that point to an opposite corner of a rectangle
- [ ] Update a text form field on the HTML page with the well known text (WKT) of the geometry that is created

**Comment by kaitlyndlee** [comment]
Does the box need to be saved when switching the projection?

**Comment by USGSsakins** [comment]
@kaitlyndlee Yes, if the user has created a geometry that needs to be saved and redrawn when the user selects a different projection.  There is a special case where they have drawn something near the equator in the simple cylindrical, then switch to a polar stereographic.  @thareUSGS What should we do if the geometry is outside the extents of the polar view?
